src/gui/optionsdialog.cpp

```diff
@@ -1689,17 +1696,24 @@ bool OptionsDialog::isSplashScreenDisabled() const
 void OptionsDialog::initializeStyleCombo()
 {
 #ifdef Q_OS_WIN
+    m_ui->comboStyle->addItem(tr("System", "System default Qt style"), u"system"_s);
```

Comment:
So is "System" item out of the sort order? Waiting for @Chocobo1 claim...

Reply:
Having it integrated into the sorted items will probably require non-clean code.
Also, IMO, it makes sense for "system" (in some sense a "default") to always be first in the list. We can drive home this intention with the usage of https://doc.qt.io/qt-6/qcombobox.html#insertSeparator
What do you think?

Reply:
> What do you think?

IMO, it is good idea to separate special item.


> Having it integrated into the sorted items will probably require non-clean code.

This should not cause problems if all the items were handled uniformly, i.e. they would have a separate display name and value stored in combobox, as you did for the System style. BTW, it would also allow to provide consistently formatted display names, i.e. "Windows", "WindowsVista", "Windows11", "Fusion", "System".

Reply:
>We can drive home this intention with the usage of https://doc.qt.io/qt-6/qcombobox.html#insertSeparator
What do you think?

I'm ok with it.